pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -291,42 +281,73 @@ public String getCrcForAllSegments(
     return getAllCrcMetadataForTable(tableName);
   }
 
-  //////////////////////////////////////////////////////////////////////////////////////
+  private List<String> getExistingTableNamesWithType(@Nonnull String tableName,
```

Comment:
Can you add the documentation for this helper function?

Reply:
Done